### DISCUSSION TITLE
pyBullet with Oculus Quest 2 on Linux

### DISCUSSION THREAD

**Discussion by JasonJZLiu** [bulletphysics/bullet3#3943] [Q&A]
Hello,

I am attempting to interface with Oculus Quest 2  in pyBullet. I was wondering if anyone has tried this on Ubuntu and could provide any guidance on setting this up?

Thanks in advance!
Jason

**Comment by ritviksingh9** [comment]
+1 I've been wondering the same thing

**Comment by BluePrintRandom** [comment]
for now there is a  experimental fork of blender with openXR and a experimental branch of upbge with openXR if one feels like tinkering.
https://builder.blender.org/download/experimental/blender-3.0.0-alpha+xr-controller-support.a063d580805c-windows.amd64-release.zip


everything is pretty much wrapped except bt_muiltibody and heightfield I think 
https://drive.google.com/open?id=14O4_htwOqN8VaeYQiUfh3AWAsIW-7VRK

to access the XR actions you bind in game check bpy.context.xr_session_state / contoller pos / rot etc.
